Source/santad/DataLayer/SNTRuleTableTest.m

```diff
@@ -533,4 +534,10 @@ - (void)testCriticalBinariesProduceFullSigningInformation {
   XCTAssertEqualObjects(teamID, cd.teamID, @"team IDs should match");
 }
 
+// This test ensures that we bump the constant on updates to the rule table
+// schema.
+- (void)testConstantVersionIsUpdated {
```

Comment:
This test wouldn't have prevented this bug. The test needs to initialize a database and check the version matches the expected version